apps/build-onchain-apps/src/components/mint/SignatureMint.tsx

```diff
@@ -36,6 +36,8 @@ const codeBlock3 = `# Edit the .env file and add the following key:
 SIGNATURE_MINT_SIGNER="WalletAddressFromAbove"
 `;
 
+const codeBlock4 = `curl 'http://localhost:3000/api/mint/signature/free?chainId=84532&wallet=0xWALLET_ADDRESS'`;
```

Comment:
Let's use ENV variable here instead of localhost, so that it works when deployed on vercel as well.

Reply:
Actually nver mind.